src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -220,6 +220,16 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(smoke_NegativeQuantizedMatMulMultiplyFusion.*)");
     // int8 specific
     retVector.emplace_back(R"(smoke_Quantized.*)");
+    retVector.emplace_back(R"(smoke_Conv_Sum_Broadcast_INT8/ConvSumInPlaceTestInt8.*)");
+    // ACL does not support blocked memory formats
+    retVector.emplace_back(R"(.*(in|out)Fmts=nC(w|hw|dhw)(8|16)c.*)");
```

Comment:
do we test blocked format in subgraph tests?
If so, we need to split thme the same way we split single layer tests

Reply:
Removed these skips and moved tests with blocked memory format to x64 folder